### PR TITLE
Add tree-sitter plugin for symbol and string collection

### PR DESCRIPTION
**Pull request by keshav-space**
This PR adds support to collect symbols and strings using tree-sitter
Reference: https://github.com/nexB/purldb/issues/388

### CODE REVIEW

**Review by pombredanne — APPROVED** [review]
Looking good!
Release time!

File: tests/test_symbols_tree_shitter.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
```
Comment (pombredanne) [inline tests/test_symbols_tree_shitter.py:1]: @keshav-space I am pretty sure you did not mean to name the file this?
test_symbols_tree_shitter.py -> test_symbols_tree_sitter.py

Comment (keshav-space) [reply tests/test_symbols_tree_shitter.py:1]: That's bad typo.

Comment (keshav-space) [reply tests/test_symbols_tree_shitter.py:1]: fixed

**Pull request merged**